tests/e2e/utils/pages/setup-ads/setup-ads-accounts.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Internal dependencies
+ */
+import MockRequests from '../../mock-requests';
+
+export default class SetupAdsAccount extends MockRequests {
+	/**
+	 * @param {import('@playwright/test').Page} page
+	 */
+	constructor( page ) {
+		super( page );
+		this.page = page;
+	}
+
+	/**
+	 * Get Continue button.
+	 *
+	 * @return {Promise<import('@playwright/test').Locator>} The Continue button.
+	 */
+	async getContinueButton() {
+		return this.page.getByRole( 'button', {
+			name: 'Continue',
+			exact: true,
+		} );
+	}
+
+	/**
+	 * Get Connect Account button.
+	 *
+	 * @return {Promise<import('@playwright/test').Locator>} The Connect button.
+	 */
+	async getConnectAdsButton() {
+		return this.page.getByRole( 'button', {
+			name: 'Connect',
+			exact: true,
+		} );
+	}
+
+	/**
+	 * Click the Continue button.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async clickContinue() {
+		await ( await this.getContinueButton() ).click();
+	}
+
+	/**
+	 * Click the Connect Ads button.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async clickConnectAds() {
+		await ( await this.getConnectAdsButton() ).click();
```

Comment:
If we remove `async` for function `getConnectAdsButton` we can rewrite this line to

```js
await this.getConnectAdsButton().click();
```